llm_studio/src/utils/utils.py

```diff
@@ -31,16 +31,11 @@ def set_seed(seed: int = 1234) -> None:
 
 def set_environment(cfg):
     """Sets and checks environment settings"""
-    if not cfg.environment.openai_api_token:
-        cfg.environment.openai_api_token = os.getenv("OPENAI_API_KEY", "")
-    if not cfg.logging.neptune_api_token:
-        cfg.logging.neptune_api_token = os.getenv("NEPTUNE_API_TOKEN", "")
-    os.environ["OPENAI_API_KEY"] = cfg.environment.openai_api_token
-    openai.api_key = cfg.environment.openai_api_token
```

Comment:
This line is no longer needed?

Reply:
We are setting it here, or?
`openai.api_key = os.getenv("OPENAI_API_KEY", "")`

Reply:
Ah yes missed the lower part